integration-tests/pipelines/konflux-e2e-tests-pipeline.yaml

```diff
@@ -45,7 +50,23 @@ spec:
       default: 'none'
       description: 'Container image built from any konflux git repo. Use this param only when you run Konflux e2e tests
         in another Konflux component repo. Will pass the component built image from the snapshot.'
+    - name: enable-sealights
+      description: "A flag to enable or disable the Sealights integration feature. When set to 'true', test results are sent to Sealights for analysis; otherwise, this feature is skipped."
+      default: "true"
   tasks:
+    - name: sealights-refs
```

Comment:
for some reason I dont see this task creating in ur pipelines

Reply:
Yes. But why?